media/css/privacy/cookie-settings-form.scss

```diff
@@ -80,6 +80,10 @@ main {
 
     .cookie-summary {
         border-bottom: 1px solid $color-light-gray-40;
+
+        &:hover {
+            cursor: pointer;
+        }
```

Comment:
I have opened an issue upstream so it can be referenced for later:
```suggestion
        // Temporary fix until https://github.com/mozilla/protocol/issues/948 is added
        &:hover {
            cursor: pointer;
        }
```